### DISCUSSION TITLE
Support consumer group

### DISCUSSION THREAD

**Discussion by jackchuong** [strimzi/strimzi-kafka-operator#9383] [Q&A]
Hi all,
I'm using Strimzi Kafka operator for creating and managing Kafka on K8S, does it support for creating and managing consumer group ?

**Comment by sreejesh123** [comment]
The process I followed is , during the Kafkauser creation we provisioned the group access and we asked consumer to use  what we provisioned in acl, when they consume  

I am not aware nor created  any object called group explicitly. Might be one exist I didn’t create

**Comment by scholzj** [comment]
Consumer groups are created by consumers directly as they subscribe as part of them. It is not something that can be controlled like topics or users. As mentioned above, you can grant the right to use a particular consumer group in the `KafkaUser` ACLs, but that is all we provide. For anything else, you have to use the Kafka Admin APi.
